analyzer-core/src/test/java/ffb/analyzer/core/queue/PriorityQueueManagerTests.java

```diff
@@ -43,13 +45,13 @@ public void performOneTask() {
     @Test
     public void performInPriorityOrder() {
```

Comment:
Thoughts on making `STARTING_VALUE + 1` a local variable you compare against?

Reply:
I like it. It's called enough times it should have it's own label.